csdp/hybrid/basic/apps/app-proxy/kustomization.yaml

```diff
@@ -3,3 +3,8 @@ kind: Kustomization
 
 components:
   - ../../../../base_components/apps/app-proxy/_bundles/basic
+
+images:
+  - name: quay.io/codefresh/cap-app-proxy
+    newName: quay.io/codefresh/cap-app-proxy
+    newTag: 1.2095.0-CR-16999-f4864d6
```

Comment:
this shouldn't be here, but there should be a version bump in `csdp/base_components/apps/app-proxy/_components/codefresh-base/kustomization.yaml`

Reply:
sure, it was just for testing, will be removed